FantasmoSDK/app/build.gradle

```diff
@@ -49,26 +51,35 @@ android {
 
     flavorDimensions "env"
     productFlavors {
+        buildTypes{
+            release{
+                buildConfigField "String", "FM_API_BASE_URL", "\"https://api.fantasmo.io/v1/image.localize\""
```

Comment:
Not sure about the naming here. The base URL should just be the host and protocol without any versioning, i.e. `https://api.fantasmo.io`

Otherwise the name `"FM_API_IMAGE_LOCALIZE_URL"` would be better suited 😸